packages/js/src/MatomoTracker.ts

```diff
@@ -234,6 +248,12 @@ class MatomoTracker {
     )
   }
 
+  // Return all ecommerce items currently in the untracked ecommerce order.
+  // https://matomo.org/docs/ecommerce-analytics
+  getEcommerceItems(): EcommerceItems {
+    return (this.callMethod('getEcommerceItems') || {}) as EcommerceItems
```

Comment:
Could we use a generic here instead of casting it? Then it would be `this.callMethod<EcommerceItems>`